content/en/security_platform/guide/how-appsec-works.md

```diff
@@ -0,0 +1,68 @@
+---
+title: How Application Security works in Datadog
+kind: guide
+---
+
+## Overview
+
+Application Security monitors application-level attacks aiming to exploit code-level vulnerabilities.
+
+APM records information about each HTTP request, these are referred to as traces. Application security uses the information APM is already collecting, and flags any suspicious requests or attack attempts as security signals. Depending on your settings and the severity of the attempt, you can receive notifications from Slack, email, or PagerDuty.
```

Comment:
attack attempt is what we detect. "suspicious" is how Application Security qualifies a request where such attempts has been detected

Reply:
> Depending on your settings and the severity of the attempt, you can receive notifications from Slack, email, or PagerDuty.

Not attack attempt level, only Security Signals can alert, as they aggregate multiple traces and detect attacks impacting production systems